server/server.js

```diff
@@ -218,15 +204,15 @@ app.post('/delete/:id', (req, res) => {
     return;
   }
 
-  storage
-    .delete(id, delete_token)
-    .then(err => {
-      if (!err) {
-        log.info('Deleted:', id);
-        res.sendStatus(200);
-      }
-    })
-    .catch(err => res.sendStatus(404));
+  try {
+   const err = await storage.delete(id, delete_token);
```

Comment:
Sorry I totally forgot it! Maybe we should add pre-commit or pre-push hook to prevent these kind of mistake.